modules/core/shared/src/main/scala/lucuma/core/validation/package.scala

```diff
@@ -34,82 +34,64 @@ package object validation {
   type ValidFormatNec[E, A, B] = ValidFormat[NonEmptyChain[E], A, B]
   type InputValidFormat[A]     = ValidFormatNec[NonEmptyString, String, A]
 
-  implicit class StringParseOps(private val s: String) extends AnyVal {
-
-    /** Try to parse as a `BigDecimal` */
-    def toBigDecimalOption: Option[BigDecimal] =
-      Try(BigDecimal(s)).toOption
-  }
-
-  implicit class EitherStringOps[A](private val e: Either[String, A]) extends AnyVal {
+  extension[A](e: Either[String, A])
 
     /** Convert an `Either[String, A]` to an `Either[Errors, A]` */
     def toEitherErrorsUnsafe: EitherErrors[A] =
       e.leftMap(s => NonEmptyChain(NonEmptyString.unsafeFrom(s)))
-  }
 
-  implicit class EitherNESOps[A](private val e: Either[NonEmptyString, A]) extends AnyVal {
+  extension[A](e: Either[NonEmptyString, A])
 
     /** Convert an `Either[NonEmptyString, A]` to an `Either[Errors, A]` */
     def toEitherErrors: EitherErrors[A] =
       e.leftMap(s => NonEmptyChain(s))
-  }
 
-  implicit class NESValidSplitEpiOps[A, B](private val self: ValidSplitEpi[NonEmptyString, A, B])
-      extends AnyVal {
+  extension[A, B](self: ValidSplitEpi[NonEmptyString, A, B])
     def toErrorsValidSplitEpi: ValidSplitEpi[Errors, A, B] =
       ValidSplitEpi(self.getValid.andThen(_.leftMap(s => NonEmptyChain(s))), self.reverseGet)
-  }
 
-  implicit class StringValidSplitEpiOps[A, B](private val self: ValidSplitEpi[String, A, B])
-      extends AnyVal {
+  extension[A, B](self: ValidSplitEpi[String, A, B])
     def toErrorsValidSplitEpiUnsafe: ValidSplitEpi[Errors, A, B] =
       ValidSplitEpi(
         self.getValid.andThen(_.leftMap(s => NonEmptyChain(NonEmptyString.unsafeFrom(s)))),
         self.reverseGet
       )
-  }
 
-  implicit class NESValidWedgeOps[A, B](private val self: ValidWedge[NonEmptyString, A, B])
-      extends AnyVal {
+  extension[A, B](self: ValidWedge[NonEmptyString, A, B])
     def toErrorsValidWedge: ValidWedge[Errors, A, B] =
       ValidWedge(self.getValid.andThen(_.leftMap(s => NonEmptyChain(s))), self.reverseGet)
-  }
 
-  implicit class StringValidWedgeOps[A, B](private val self: ValidWedge[String, A, B])
-      extends AnyVal {
+  extension[A, B](self: ValidWedge[String, A, B])
     def toErrorsValidWedgeUnsafe: ValidWedge[Errors, A, B] =
       ValidWedge(
         self.getValid.andThen(_.leftMap(s => NonEmptyChain(NonEmptyString.unsafeFrom(s)))),
         self.reverseGet
       )
-  }
 
-  implicit class InputValidWedgeOps[A](private val self: InputValidWedge[A]) extends AnyVal {
+  extension[A](self: InputValidWedge[A])
     def withErrorMessage(msg: NonEmptyString): InputValidWedge[A] =
       self.withError(NonEmptyChain(msg))
 
     /**
      * Build an `InputValidWedge[NonEmptyList[A]]` given a `InputValidWedge[A]`
      */
-    def toNel(separator: NonEmptyString = ",".refined): InputValidWedge[NonEmptyList[A]] =
+    def toNel(separator: NonEmptyString): InputValidWedge[NonEmptyList[A]] =
```

Comment:
I had to remove the default param or it would collide with the other `toNel` method. Is this a problem?

Reply:
Which other `toNel`?

Reply:
I'm guessing the one for `InputValidSplitEpi`.

@cquiroz It's not a problem but it's kind of nice to have to pass the (refined) parameter when it's a comma. What if you use `@targetName` ?

Reply:
It doesn't make a difference to have a targetName